src/test/java/io/supertokens/test/passwordless/PasswordlessConsumeCodeTest.java

```diff
@@ -740,27 +756,32 @@ public void testConsumeWrongUserInputCodeExceedingMaxAttemptsWithConfigUpdate()
         Exception error = null;
         int maxCodeInputAttempts = Config.getConfig(process.getProcess()).getPasswordlessMaxCodeInputAttempts();
 
-        // input wrong code max attempts - 2 times
-        for (int counter = 0; counter < maxCodeInputAttempts - 2; counter++) {
+        // input wrong code max attempts - 1 times
+        for (int counter = 0; counter < maxCodeInputAttempts - 1; counter++) {
 
+            Exception exception = null;
             try {
                 Passwordless.consumeCode(process.getProcess(), createCodeResponse.deviceId,
                         createCodeResponse.deviceIdHash, "n0p321", null);
             } catch (Exception ex) {
-
+                exception = ex;
             }
 
+            assertNotNull(exception);
+            assert (exception instanceof IncorrectUserInputCodeException);
+
         }
 
         // verify that devices have not been cleared
         PasswordlessDevice[] devices = storage.getDevicesByEmail(EMAIL);
         assertNotEquals(0, devices.length);
 
         // kill process and restart with increased max code input attempts
-        killApplication(process);
+        process.kill();
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STOPPED));
+        ;
 
-        String[] args = { "../" };
-        Utils.setValueInConfig("passwordless_max_code_input_attempts", "7");
+        Utils.setValueInConfig("passwordless_max_code_input_attempts", String.valueOf(maxCodeInputAttempts - 3));
```

Comment:
Does this test fail if you comment this out? If not please tweak the try counts.

Reply:
yes, the test fails when I comment it out.